ads/google/a4a/test/test-google-ads-a4a-config.js

```diff
@@ -156,7 +158,7 @@ describe('a4a_config', () => {
   it('should attach expt ID and return true when expt is on', () => {
     rand.returns(0.75);  // Random value to select the 2nd branch
     expect(googleAdsIsA4AEnabled(win, element, EXP_ID,
-        EXTERNAL_BRANCHES, INTERNAL_BRANCHES),
+        EXTERNAL_BRANCHES, INTERNAL_BRANCHES, MEASURE_ON_RENDER_BRANCH),
```

Comment:
Old comment is hidden since I changed code, but: let me know if the other files are more what you're looking for now, and in parallel I will add new unit test(s) here.